src/main/resources/PowerShell/SqlCi.ps1

```diff
@@ -471,7 +471,7 @@ function TestDatabase
 
     $connectionOptions = @{
     }
-    if (AreConnectionOptionsHandled($temporaryDatabaseEncryptConnection, $temporarTestSourceOptionyDatabaseTrustServerCertificate)){
+    if (AreConnectionOptionsHandled($temporaryDatabaseEncryptConnection, $temporaryDatabaseTrustServerCertificate)){
         $connectionOptions += @{ 'Encrypt' = $temporaryDatabaseEncryptConnection.ToBool() }
```

Comment:
Is the ToBool necessary here, or should the above arguments also be converted. Not sure if AreConnectionOptionsHandled would work correctly if they're not bool going in.